hrident/info.xml

```diff
@@ -10,11 +10,10 @@
   </maintainer>
   <releaseDate>2017-08-22</releaseDate>
```

Comment:
Is this release date correct or should it be `2018-03-26` since the `develStage` has been updated?

Reply:
The date is correct. This extension has not been updated in a while. In fact, it will soon be deleted, as it's not installed anymore on new sites.

Reply:
Great.